config/deploy/server_settings.rb

```diff
@@ -1,31 +1,9 @@
 # Note: this file is required in config/deploy/<environment>.rb
 
 puts "ENV['APP_ENV']  = #{ENV['APP_ENV']}"
-puts "ENV['SDR_APP']  = #{ENV['SDR_APP']}"
-puts "ENV['SDR_USER'] = #{ENV['SDR_USER']}"
-puts "ENV['SDR_HOST'] = #{ENV['SDR_HOST']}"
-puts "ENV['SDR_URL']  = #{ENV['SDR_URL']}"
 puts
 
-set :default_env, {
-    'APP_ENV'  => ENV['APP_ENV'],
-    'SDR_APP'  => ENV['SDR_APP'],
-    'SDR_USER' => ENV['SDR_USER'],
-    'SDR_HOST' => ENV['SDR_HOST'],
-    'SDR_URL'  => ENV['SDR_URL'],
-}
-
-server ENV['SDR_HOST'], user: ENV['SDR_USER'], roles: %w{app}
-Capistrano::OneTimeKey.generate_one_time_key!
-ssh_opts = fetch(:ssh_options)
-ssh_opts[:forward_agent] = true
-ssh_opts[:verbose] = false
-# The :ssh_options are set by
-# capistrano-one_time_key/blob/master/lib/capistrano/tasks/one_time_key.rake
-
-# Target path
-USER_HOME = `ssh #{ENV['SDR_USER']}@#{ENV['SDR_HOST']} 'echo $HOME'`.chomp
-set :deploy_to, "#{USER_HOME}/#{ENV['SDR_APP']}"
+set :default_env, 'APP_ENV'  => ENV['APP_ENV']
```

Comment:
With the refactor in this PR, the `APP_ENV` will not longer have a default value.  Please ensure that this gets a default value, as in something like this `ENV['APP_ENV']  ||= 'development'`.  Also note that all the `sdr-configs` files had specific settings for this ENV.  I _guess_ these values could go into a rails-config Settings value instead, if they are not deployment values.  I'm trying to recall why it's set and I _guess_ it might have something to do with the groups defined for bundler.